pkgs/by-name/cr/cryptpad/package.nix

```diff
@@ -120,8 +162,9 @@ buildNpmPackage {
     # Move to install directory manually.
     npm run install:components
     mv www/components "$out_cryptpad/www/"
-    # and fix absolute symlink to /build...
-    ln -Tfs ../../src/tweetnacl "$out_cryptpad/www/components/tweetnacl"
+    # Remove the directory first if it exists, then create the symlink
```

Comment:
I don't think we need to bother with this anymore in this version?
In 2025.3.0 www/components/tweetnacl was a symlink to /build so we had to fix it, but in the new version the directory is properly installed

OTOH the content is identical so symlink is probably fine (so long as cryptpad doesn't do any weird check based on realpath), just that at this point I've checked and all directories in www/components also exist and are identical to node modules so if we do that we might just as well replace them all and not single tweetnacl out
(nix store optimize will fix all the individual files away, but it's 225MB so I assume it'd be better optimized away if it doesn't hurt)